features/configuration_files/directory_specific_directives.feature

```diff
@@ -163,11 +166,13 @@ Feature: Directory directives
     When I run `reek -c config.reek other/ web_app/`
     Then it reports:
     """
-    other/projects_controller.rb -- 1 warning:
+    other/projects_controller.rb -- 2 warnings:
+      [1]:InstanceVariableAssumption: ProjectController assumes too much for instance variable @project [https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md]
       [4]:NestedIterators: ProjectController#show contains iterators nested 2 deep [https://github.com/troessner/reek/blob/master/docs/Nested-Iterators.md]
-    web_app/app/controllers/users_controller.rb -- 1 warning:
+    web_app/app/controllers/users_controller.rb -- 2 warnings:
+      [1]:InstanceVariableAssumption: UsersController assumes too much for instance variable @user [https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md]
       [1]:IrresponsibleModule: UsersController has no descriptive comment [https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md]
-    2 total warnings
+    4 total warnings
```

Comment:
Can we rather update the code samples for this features so that it doesn't smell of this, meaning this feature and stdin.feature dont show up at all in this PR?